sdk/search/Microsoft.Azure.Search.Common/src/Microsoft.Azure.Search.Common.csproj

```diff
@@ -1,15 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSearch.DataPlane.props'))" />
   <PropertyGroup>
-    <PackageId>Microsoft.Azure.Search.Common</PackageId>
+    <PackageTags>Microsoft Azure Search;Search</PackageTags>
```

Comment:
Is the package ID derived from the project file name now?

Reply:
Yes it is defaulted from the project name.

Reply:
So let's say we decide to solve our GA-versus-preview problem by co-locating the code instead of maintaining a separate branch -- If we need to create a new project with a different project name, but the same package name, can we still use `PackageId` to accomplish that?

Reply:
the project name is the default for the package id but you can still explicitly set it if needed. However given that it is redundant here it is best to eliminate it to simply the project and prevent potential issues.

Reply:
Agreed; Thanks for clarifying.